### PR TITLE
add link to rust docs

### PR DESCRIPTION
**Pull request by Roznovjak**

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ module.exports = {
     Build: [
       "infra_testnet"
     ],
+    Docs: [
```
Comment (vgantchev) [inline sidebars.js:19]: I think it is confusing to have a section called "Docs" in the docs 

can you put it as a direct link `Rust Docs` instead? What I mean is without the expandable dropdown

Comment (Roznovjak) [reply sidebars.js:19]: I tried, but I couldn't find a way how to achieve this result

Comment (vgantchev) [reply sidebars.js:19]: alternatively you can move it as first link under section Build

**Comment by vgantchev** [comment]
sry for the late comment, but I would suggest to move the rust docs above the infra stuff :)

**Review by vgantchev — APPROVED** [review]

**Pull request merged**